specification/storagepool/resource-manager/Microsoft.StoragePool/preview/2021-04-01-preview/examples/IscsiTargets_Put.json

```diff
@@ -0,0 +1,69 @@
+{
+  "parameters": {
+    "api-version": "2021-04-01-preview",
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "resourceGroupName": "myResourceGroup",
+    "diskPoolName": "myDiskPool",
+    "iscsiTargetName": "myIscsiTarget",
+    "iscsiTargetCreatePayload": {
+      "properties": {
+        "targetIqn": "iqn.2005-03.org.iscsi:server1",
+        "luns": [
+          {
+            "managedDiskAzureResourceId": "/subscriptions/11111111-1111-1111-1111-111111111111/resourceGroups/myResourceGroup/providers/Microsoft.Compute/disks/vm-name_DataDisk_1",
+            "name": "lun0"
+          }
+        ],
+        "aclMode": "Dynamic"
+      }
+    }
+  },
+  "responses": {
+    "202": {
```

Comment:
>"202":  [](start = 4, length = 7)

202 should return Azure-AsyncOperation or Location header without body. The recommended pattern is 201 + Provisioningstate. + Azure-AsyncOperation header.